plugins/telemetry-otel/src/internalClusterTest/java/org/opensearch/telemetry/metrics/TelemetryMetricsEnabledUpDownCounterIT.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry.metrics;
+
+import org.opensearch.common.settings.Settings;
+import org.opensearch.common.unit.TimeValue;
+import org.opensearch.plugins.Plugin;
+import org.opensearch.telemetry.IntegrationTestOTelTelemetryPlugin;
+import org.opensearch.telemetry.OTelTelemetrySettings;
+import org.opensearch.telemetry.TelemetrySettings;
+import org.opensearch.test.OpenSearchSingleNodeTestCase;
+import org.junit.After;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+
+import io.opentelemetry.sdk.metrics.data.DoublePointData;
+
+public class TelemetryMetricsEnabledUpDownCounterIT extends OpenSearchSingleNodeTestCase {
```

Comment:
We should combine these test cases `under TelemetryMetricsEnabledSanityIT` (or something), the metrics are not disabled by type

Reply:
Make sense. It's possible now to write more tests with predictable results in single class after adding the reset method.